.github/workflows/ci.yml

```diff
@@ -30,6 +30,17 @@ jobs:
         npm run format:check --prefix src/hypofuzz/frontend
 
   docs:
+    if: >
```

Comment:
looks like this is invalid syntax? https://github.com/Zac-HD/hypofuzz/actions/runs/14133098296/workflow#L33

Reply:
ack, I was wondering why the checks on this PR were waiting indefinitely, unfortunate ux that this is only accessible from the actions tab